### ISSUE TITLE
Cursor is auto selecting text after toggling fullscreen in macOS

### ISSUE THREAD

**Issue by ooJerryLeeoo** [jwilm/alacritty#1246]
**Which operating system does the issue occur on?** macOS 10.13.4

Problem as shown in the title, here's how to reproduce:

1. Open Alacritty
2. Click the green full screen button to enter fullscreen
3. Moving the cursor around ***without holding left mouse button***
4. And the selection is following the cursor

the same problem also happens when exit fullscreen

Here's a demonstration:

![cursor is auto selecting text after toggling fullscreen](https://user-images.githubusercontent.com/20783502/38736874-0b8c7262-3f60-11e8-9d2c-15e884fe301c.gif)

**Comment by chrisduerr** [comment]
This has been mentioned in #362 before. But it's probably nice to have a separate issue for this.

**Comment by ooJerryLeeoo** [comment]
This also happens after resizing the window:

![kapture 2018-04-14 at 6 57 48](https://user-images.githubusercontent.com/20783502/38760890-4b1d89ce-3fb1-11e8-92de-c81e892acaa9.gif)

**Comment by sodiumjoe** [comment]
This appears to be a bug in `winit`. In winit's window example, there's a MouseInput event after resizing. I've made an issue there: https://github.com/tomaka/winit/issues/463

**Comment by sodiumjoe** [comment]
There's a PR for a fix for this: https://github.com/tomaka/winit/pull/466, so we're just waiting on that to merge, release, and propagate up through glutin.

**Comment by sodiumjoe** [comment]
I think this is actually basically a dupe of #47

**Comment by chrisduerr** [comment]
Closing as duplicate.
